packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -1321,7 +1326,7 @@ class SvgParser {
     }
 
     if (rawFill.startsWith('url')) {
-      return SvgFillAttributes._(
+      return SvgFillAttributes(
```

Comment:
I'm not a huge fan of making the ctors of these public. They're challenging to get right and in flutter_svg this kind of thing was a source of bugs where multiple parts of the library would differently and incorrectly construct attribute lists.

Reply:
I may not have read the code far enough yet, but instead it might work to have methods that construct new attribute objects with the appropriate changes/merges...

Reply:
I would really like to be able to unit test the visitors...

Reply:
Right but can we write unit tests on them from simplified SVG strings that have the right attributes?

Reply:
Yeah will do